### ISSUE TITLE
[SUGGESTION] Add Industrial Lights/Lamps

### ISSUE THREAD

**Issue by drenough** [TinyModularThings/IC2Classic#825]
Hi, it would be really nice addition if you could add Industrial Lights/Lamps, non inverted versions and inverted versions, but in Industrial theme/style. I know there is "Construction Light" but it would be really amazing if we could have more lights/lamps in industrial theme/style.

**Comment by Speiger** [comment]
@drenough I asked specifically asked for ideas like that during the beta, but nothing was suggested what could be done.
Soooo nothing was ever added.
I would love to throw this on the todolist for the next "Major update" of the mod. If you have examples I would make a record of them.

But the next major version will be on a new mc version.

**Comment by drenough** [comment]
Oh, my bad, I wasn't around during beta :(. Anyway, I dont have any example for texture rn, but I could try make some later if that's ok ?

**Comment by Speiger** [comment]
@drenough yeah its fine, it was a closed beta with like 15-20 people anyways. Its harder to do proper "Betas" with more then that many people. (That was 6 months of daily patches)     

Also yeah that would be nice.

**Comment by drenough** [comment]
Version one of yellow illumar lamp on and off. Working on 2 more examples rn
![yellow_illumar_lamp_on_V1](https://github.com/TinyModularThings/IC2Classic/assets/55913955/2e762260-f6f8-42b9-8637-8b565020f135)
![yellow_illumar_lamp_off_V1](https://github.com/TinyModularThings/IC2Classic/assets/55913955/38103d0d-b1b5-4363-b64a-8a958a0f77f4)

**Comment by drenough** [comment]
Would this be too much as reinforced yellow lamp ? I used colors from reinforced block for edges so it's actually looking good and fit the reinforced theme
But the non reinforced version would have more like grey/white/black kind of vibe for edges so it's different
This is just idea, so keep that in mind 🙂
![yellow_illumar_lamp_off_V2 test](https://github.com/TinyModularThings/IC2Classic/assets/55913955/67257110-6db9-4b20-ab75-d19ba8c849a9)

**Comment by drenough** [comment]
This is the reinforced yellow lamp on V2 (without corner changes, gonna do them now)
![yellow_illumar_lamp_on_V2 test](https://github.com/TinyModularThings/IC2Classic/assets/55913955/0a1429af-951d-423e-b9eb-926188d9af96)

**Comment by drenough** [comment]
This is the reinforced yellow lamp on V3 (with corner changes)
![reinforced_yellow_illumar_lamp_on_V3](https://github.com/TinyModularThings/IC2Classic/assets/55913955/ead6357f-547d-40aa-baaa-1995e1bbe5e0)

**Comment by drenough** [comment]
![reinforced_yellow_illumar_lamp_off_V3](https://github.com/TinyModularThings/IC2Classic/assets/55913955/cc98b9d7-0b6d-4402-99b7-f9bd6d452572)

**Comment by drenough** [comment]
Here are textures, if you want source file for blockbench, just DM me on discord and I can send that over
![reinforced_yellow_illumar_lamp_off_V3](https://github.com/TinyModularThings/IC2Classic/assets/55913955/cdf6bbfb-4ee2-4a77-b0ab-07a8303a0c3a)
![reinforced_yellow_illumar_lamp_on_V3](https://github.com/TinyModularThings/IC2Classic/assets/55913955/dac81441-759b-4f23-bf5c-63de392bcc1d)
